apps/whale-api/src/module.api/governance.service.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  ListProposalsStatus,
+  ListProposalsType,
+  ProposalInfo,
+  ProposalStatus,
+  ProposalType,
+  VoteResult
+} from '@defichain/jellyfish-api-core/dist/category/governance'
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+import { BadRequestException, Injectable } from '@nestjs/common'
+import { ApiPagedResponse } from './_core/api.paged.response'
+import { PaginationQuery } from './_core/api.query'
+import { NotFoundApiException } from './_core/api.error'
+import {
+  GovernanceProposal,
+  GovernanceProposalStatus,
+  GovernanceProposalType,
+  ProposalMasternodeType,
+  ProposalVoteResultType,
+  ProposalVotesResult
+} from '@defichain/whale-api-client/dist/api/governance'
+
+@Injectable()
+export class GovernanceService {
+  constructor (private readonly client: JsonRpcClient) {}
+
+  async list (
```

Comment:
```suggestion
  async listGovernanceProposal (
```

Reply:
Unless we call this service `GovernanceProposalService`

Reply:
updated the method name, not renaming the file for now, will KIV if more methods are added in the future